smart_control/notebooks/SAC_Demo.ipynb

```diff
@@ -991,137 +991,13 @@
-            "  np.array(weekend_energy_prices)\n",
-            "/home/trigo/sbsim/sbsim/smart_control/simulator/building_utils.py:283: UserWarning: Connected components is showing that there are 4 or fewer\n",
-            "     rooms in your building. You may have your 0's and 1's inverted in the\n",
-            "     floor_plan. Remember that for the connectedComponents function,\n",
-            "     0's must code for exterior space and exterior or interior walls,\n",
-            "     and 1's must code for interior space.\n",
-            "  warnings.warn(\"\"\"Connected components is showing that there are 4 or fewer\n",
-            "/home/trigo/sbsim/sbsim/smart_control/reward/electricity_energy_cost.py:147: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.\n",
-            "  np.array(carbon_emission_rates) / 1.0e6 / 3600.0\n",
-            "/home/trigo/sbsim/sbsim/smart_control/reward/electricity_energy_cost.py:152: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.\n",
-            "  np.array(weekday_energy_prices)\n",
-            "/home/trigo/sbsim/sbsim/smart_control/reward/electricity_energy_cost.py:159: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.\n",
-            "  np.array(weekend_energy_prices)\n",
-            "/home/trigo/sbsim/sbsim/smart_control/simulator/building_utils.py:283: UserWarning: Connected components is showing that there are 4 or fewer\n",
-            "     rooms in your building. You may have your 0's and 1's inverted in the\n",
-            "     floor_plan. Remember that for the connectedComponents function,\n",
-            "     0's must code for exterior space and exterior or interior walls,\n",
-            "     and 1's must code for interior space.\n",
-            "  warnings.warn(\"\"\"Connected components is showing that there are 4 or fewer\n",
-            "/home/trigo/sbsim/sbsim/smart_control/reward/electricity_energy_cost.py:147: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.\n",
-            "  np.array(carbon_emission_rates) / 1.0e6 / 3600.0\n",
-            "/home/trigo/sbsim/sbsim/smart_control/reward/electricity_energy_cost.py:152: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.\n",
-            "  np.array(weekday_energy_prices)\n",
-            "/home/trigo/sbsim/sbsim/smart_control/reward/electricity_energy_cost.py:159: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.\n",
-            "  np.array(weekend_energy_prices)\n"
-          ]
-        }
-      ],
+      "outputs": [],
       "source": [
-        "@gin.configurable\n",
-        "def to_timestamp(date_str: str) -> pd.Timestamp:\n",
```

Comment:
Confirm I am running into the same issue about ambiguous functions (`to_timestamp`, etc). I'm going to replicate removing these function definitions (or maybe just commenting them out) in #23

Reply:
JK I want to prioritize merging this PR first, before #23